examples/webgpu_video_frame.html

```diff
@@ -71,14 +78,33 @@
 						decoder.decode( chunk );
 
 					},
-					setStatus( s ) {
+					setStatus( kind, status ) {
+
+						console.info( 'MP4Demuxer:', kind );
+
+						if ( kind === 'fetch' && status === 'Done' ) {
+
+							decoder.flush().then( () => {
+
+								decoder.close();
+
+								scene.remove( mesh );
+								geometry.dispose();
+								material.dispose();
+								videoFrameTexture.dispose();
+
```

Comment:
Let's simplify this a bit since all the `dispose()` calls feel like over-engineering. The demo does not consistently add and remove stuff it is sufficient if you just close the decoder and the video frame.

Reply:
simplified